<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>654</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>969</width>
    <height>654</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Главное окно</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>751</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="WidgetPlotParametr"/>
   </widget>
   <widget class="QPushButton" name="ButWriteOnOff">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>450</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>Вкл. запись</string>
    </property>
   </widget>
   <widget class="QComboBox" name="BoxSetParametrToPlot">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>450</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Частота вращения ротора КНД</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Частота вращения ротора КВД</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Температура газа за КС</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Температура газа за ТНД</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Давление за КВД</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Тяга</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Газодинамическая устойчивость</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>413</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать параметр</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>450</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Запись в файл</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="EnableSavingByDate">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>420</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Включить сохранение по дате</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>COM-порт</string>
    </property>
    <widget class="QPushButton" name="ButExchange">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Начать обмен</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>961</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Сообщения</string>
    </property>
    <widget class="QListWidget" name="ListMessage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>911</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ButClearListessage">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>90</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: outset; border-width: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/clear.png</normaloff>images/clear.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>190</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="title">
     <string>Режим моделирования</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>40</y>
       <width>163</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="spinBoxRUD">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxT0">
        <property name="minimum">
         <double>-273.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>288.149999999999977</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxP0">
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>1.030000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>19</y>
       <width>171</width>
       <height>18</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelRUD">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;РУД&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>T*вх</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>P*вх</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="ClearGraphic">
    <property name="geometry">
     <rect>
      <x>841</x>
      <y>414</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить график</string>
    </property>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>ButWriteOnOff</zorder>
   <zorder>BoxSetParametrToPlot</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>EnableSavingByDate</zorder>
   <zorder>ClearGraphic</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Открыть</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_COM"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>График из файла</string>
   </property>
  </action>
  <action name="actiondd">
   <property name="text">
    <string>dd</string>
   </property>
  </action>
  <action name="actiondfef">
   <property name="text">
    <string>dfef</string>
   </property>
  </action>
  <action name="action_COM">
   <property name="text">
    <string>Настройки COM-порта</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
